app/scripts/controllers/preferences.js

```diff
@@ -155,6 +166,21 @@ class PreferencesController {
     return this.store.getState().tokens
   }
 
+  /**
+   * Sets a custom label for an account
+   * @param {string} account the account to set a label for
+   * @param {string} label the custom label for the account
+   * @return {Promise<string>}
+   */
+  async setAccountLabel (account, label) {
```

Comment:
It doesn't but the similar function in `KeyringController` used to be, so I kept it `async` for compatibility

Reply:
Since you're already returning a Promise, I don't think marking this as `async` without using `await` statements within the function body actually does anything.